doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -202,14 +202,15 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
 ----
 
-``--cpu_affinity <list of pCPUs>``
-   list of pCPUs assigned to this VM.
+``--cpu_affinity <list of lapic_id>``
+   list of vCPUs assigned to this VM, the vCPUs are identified by their lapic IDs.
+   Lapic IDs can use ``apicid`` in Service VM's ``/proc/cpuinfo``.
```

Comment:
I think this needs more information.  What's the difference between apicid and lapic_id?

Reply:
lapic_id is short for 'local apic id', is consistent with SDM , and is more precise. 
'apicid' is the description in ``/proc/cpuinfo`` in linux OS.

Do you have some suggestion for more information?

Reply:
Updated， please check if it's easier to understand.